### PR TITLE
config-generator: CNF Installation (2.2) Add generator for v2 config

### PR DESCRIPTION
**Pull request by kosstennbl**
## Description
Create new config generation task, that would generate v2 config.
This task should be more interactable and friendly to new testsuite users.

## Issues:
Refs: #2138

### CODE REVIEW

**Comment by kosstennbl** [comment]
Can be merged only after #2129

**Comment by svteb** [comment]
Perhaps change the title so it conforms to the other epic tasks so far?

File: src/tasks/cnf_setup.cr [hunk 1]
```diff
@@ -99,3 +104,69 @@ end
...
+
+  loop do
+    puts "Select deployment type:"
+    puts "1. Helm Chart"
+    puts "2. Helm Directory"
+    puts "3. Manifest Directory"
+    puts "4. Finish and save configuration"
+    print "Enter your choice (1-4): "
+    
+    choice = gets.try(&.strip)
```
Comment (svteb) [inline src/tasks/cnf_setup.cr:126]: Why does this not use the `prompt` function?

Comment (kosstennbl) [reply src/tasks/cnf_setup.cr]: Fixed

**Review by svteb — APPROVED** [review]
I presume that the changes from #2129 that have been copied into this pull request will be removed before merging. Nevertheless, works as a good quick-start implementation, lgtm.

**Comment by kosstennbl** [comment]
Re-created from different branch as #2148

**Pull request closed without merging**